test/Autofac.Integration.AspNetCore.Multitenant.Test/TestDependencies/TestServerFixture.cs

```diff
@@ -79,6 +82,7 @@ public void Configure(IApplicationBuilder builder)
                         var tenantAccessor = context.RequestServices.GetRequiredService<ITenantAccessor>();
                         var strategy = context.RequestServices.GetRequiredService<ITenantIdentificationStrategy>();
 
+                        // Assert.Equal(scopedDependency, scopedDependencyB);
```

Comment:
If this isn't doing anything, can we remove the comment?

Reply:
Addressed.